mongo/integration/index_view_test.go

```diff
@@ -73,31 +73,36 @@ func TestIndexView(t *testing.T) {
 			})
 		})
 		mt.Run("all options", func(mt *mtest.T) {
+			opts := options.Index().
+				SetBackground(false).
+				SetExpireAfterSeconds(10).
+				SetName("a").
+				SetSparse(false).
+				SetUnique(false).
+				SetVersion(1).
+				SetDefaultLanguage("english").
+				SetLanguageOverride("english").
+				SetTextVersion(1).
+				SetWeights(bson.D{}).
+				SetSphereVersion(1).
+				SetBits(2).
+				SetMax(10).
+				SetMin(1).
+				SetPartialFilterExpression(bson.D{}).
+				SetStorageEngine(bson.D{
+					{"wiredTiger", bson.D{
+						{"configString", "block_compressor=zlib"},
+					}},
+				})
+
+			// Only check SetBucketSize if version is less than or equal to 4.4
+			if mtest.CompareServerVersions(mtest.ServerVersion(), "4.9") <= 0 {
```

Comment:
SERVER-47081 was closed with a fix version of 4.9.0, so setting `bucketSize` against a 4.9 server will fail, so should this be `<` rather than `<=`? I can confirm that this test fails locally against the "latest" server variant (version `4.9.0-alpha-1023-g6019169`) but passes with this suggested change.

Reply:
done